__tests__/wallet/address.spec.ts

```diff
@@ -4,10 +4,15 @@ jest.setTimeout(300000);
 
 describe('Get Address', () => {
   it('should return wallet address', async () => {
-    const { key } = await createWallet();
+    const { key } = await createWallet({
+      environment: 'local',
+    });
 
     if (key) {
-      const address = await getAddress(key);
+      const address = await getAddress({
+        key,
+        environment: 'local',
```

Comment:
The `getAddress` function does not need an environment variable. The keys and addresses are applicable the same way on both `local` and `mainnet`.